README.md

```diff
@@ -171,10 +196,11 @@ BUMP_LEVEL=minor make release
 On certain versions of the tag hardware, the data from hydrophones is stored in raw format. Convert it before upload with the following script:
 ```console
 #!/bin/bash
-OUTDIR="../flac"
+OUTDIR="../flac/${PWD##*/}"
 mkdir -p "$OUTDIR"
 for f in *.raw; do
-  flac --channels=3 --bps=16 --sample-rate=96000 --sign=signed --endian=big --force-raw-format "$f" --force --output-name="$OUTDIR/$f.flac" &
+  IFS='.' read -r -a split_filename <<< "$f"
```

Comment:
I recommend $(realpath "$PWD") instead of '.'

Reply:
Tried this and it is not quite giving me the results I want.

Previously this script was making .raw.flac files. This is part of the solution to eliminate this. I am just trying to separate the timestamped filename into 2 strings based on the dot delimiter '.'
This way I can just reference the first string in the returned list when I do ```--output-name=$OUTDIR/${split_filename[0]}.flac"```